build_all_images.sh

```diff
@@ -242,91 +242,6 @@ function build_jar_runner_image {
 	fi
 }
 
-function build_jdk11_image {
```

Comment:
@Wykenz, were you able to test locally the building of all jdk images after this refactoring?

If not, can you please do it before send the next pull?

Thanks :smile: